js/main.js

```diff
@@ -0,0 +1,59 @@
+const myBookList = [];
```

Comment:
- [ ] According to the project requirements, the library should contain some default books when the user first visits it, please populate it with at least three books of your choice.

Reply:
Thanks. I've modified it.